gss-cb-signalling.xml

```diff
@@ -154,7 +154,7 @@ GSS_S_FAILURE indicates a general failure.</t>
           </list>
         </t>
         <t>
-This function tells the mechanism (when one is eventually chosen and invoked) that the application requests the given req_flags and is prepared to check the flags in given ret_flags_understood. Mechanisms SHOULD NOT limit flags returned to those in ret_flags_understood, but MAY alter behavior accordingly. Initiators can override the req_flags in their GSS_Init_sec_context() call, but if no flags are requested there then the req_flags set on the empty context will be used.</t>
+This function tells the mechanism (when one is eventually chosen and invoked) that the application requests the given req_flags and is prepared to check the flags in given ret_flags_understood. Mechanisms SHOULD NOT limit flags returned to those in ret_flags_understood, but MAY alter behavior accordingly. Initiators can override the req_flags in their GSS_Init_sec_context() call, but if no flags are requested there then the req_flags set on the empty context will be used. Acceptors can use the req_flags parameter only to request channel bindings.</t>
```

Comment:
Wait, what?  This document doesn't say anything about using req_flags to request channel bindings; it only talks about using ret_flags_understood.

I'm trying to remember what we concluded on voice about acceptors and req_flags.  I think it was that no particular acceptor behavior on acceptor req_flags is being specified here, but that either new mechanism specifications or new flag specifications might specify behavior for acceptor req_flags in the future.  (Most likely new flag specifications.)

Reply:
Hmm, I don't remember any conversation about that.  I read it as an open question in the kitten review.  Will fix in a moment.